config/prompts_sys.py

```
@@ -265,6 +265,113 @@ def _loc(d: dict, lang: str) -> str:
+}
+
+# 无窗口标题时的 system prompt（水印后拼接）
+VISION_SYSTEM_NO_TITLE = {
+    'zh': '请简洁地描述图片中的主要内容、关键细节和你觉得有趣的地方。你的回答不能超过250字。',
+    'en': 'Briefly describe the main content, key details, and anything you find interesting in the image. Your response should not exceed 250 words.',
+    'ja': '画像の主な内容、重要な詳細、興味深い点を簡潔に説明してください。回答は250文字以内にしてください。',
+    'ko': '이미지의 주요 내용, 핵심 세부사항, 흥미로운 점을 간결하게 설명하세요. 답변은 250자를 넘지 마세요.',
+    'ru': 'Кратко опишите основное содержимое изображения, ключевые детали и интересные моменты. Ответ не должен превышать 250 слов.',
+}
+
+# 有窗口标题时的 user prompt（{window_title} 占位符）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)